### PR TITLE
[fix](Row store) get the correct value of `row_store_page_size` after FE restarts

### PR DESCRIPTION
**Pull request by lxr599**
## Proposed changes

get the correct value of `row_store_page_size` after FE restarts

### CODE REVIEW

**Comment by lxr599** [comment]
run buildall

**Comment by lxr599** [comment]
run buildall

**Comment by lxr599** [comment]
run external

**Comment by lxr599** [comment]
run buildall

**Comment by lxr599** [comment]
run external

**Comment by lxr599** [comment]
run external

**Comment by lxr599** [comment]
run external

File: be/src/olap/rowset/segment_v2/segment_writer.cpp [hunk 1]
```diff
@@ -258,8 +258,13 @@ Status SegmentWriter::_create_column_writer(uint32_t cid, const TabletColumn& co
 
     if (column.is_row_store_column()) {
         // smaller page size for row store column
-        opts.data_page_size = _tablet_schema->row_store_page_size();
+        auto page_size = _tablet_schema->row_store_page_size();
+        opts.data_page_size = (page_size > 0) ? page_size : ([this]() {
+            _tablet_schema->set_row_store_page_size(segment_v2::ROW_STORE_PAGE_SIZE_DEFAULT_VALUE);
+            return segment_v2::ROW_STORE_PAGE_SIZE_DEFAULT_VALUE;
```
Comment (eldenmoon) [inline be/src/olap/rowset/segment_v2/segment_writer.cpp:264]: don't set use default directly

**Comment by lxr599** [comment]
run buildall

**Comment by lxr599** [comment]
run buildall

**Comment by lxr599** [comment]
run p0

**Comment by lxr599** [comment]
run p0

**Comment by lxr599** [comment]
run buildall

**Review by eldenmoon — APPROVED** [review]
LGTM

**Review by xiaokang — APPROVED** [review]
LGTM

**Pull request merged**